apis/python/tests/test_basic_anndata_io.py

```diff
@@ -47,7 +47,11 @@ def adata(h5ad_file):
         ["resume"],
     ],
 )
-def test_import_anndata(adata, ingest_modes):
+@pytest.mark.parametrize(
+    "sparsify_X",
+    [True, False],
+)
+def test_import_anndata(adata, ingest_modes, sparsify_X):
```

Comment:
Given that this tests `from_anndata`, which is an in-memory format that can be read from multiple file formats (zarr, h5py, etc) and in multiple modes (backed, memory-only, etc), it would be good to expand the test matrix to include more than the single in-memory H5AD. 

See comment in `ingest.py` for how this may fail in some scenarios.

Reply:
#1064